src/lammpsio/snapshot.py

```diff
@@ -217,11 +260,25 @@ def to_hoomd_gsd(self, type_map=None):
             A map from the :attr:`Snapshot.typeid` to a HOOMD type.
             If not specified, the typeids are used as the types.
 
+            .. deprecated:: 0.7.0
+                Use `Snapshot.type_label` instead.
```

Comment:
I added this here because I saw we had a deprecation warning. We should review the API before finalizing, and mark anything else that's deprecated in the docs and with a warning.

Reply:
I searched the code and the only spot that a DeprecationWarning pops up is here!

Reply:
Are there any other parts of the code that are effectively deprecated, but a warning is not raised though? I'm thinking of things related to types, especially.

Reply:
I don't think so. I think we only had types implemented before `type_label` in `to_hoomd_gsd` where this warning is. 

Do you have something specific in mind to check?